modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/pages/questions/Questions.es.js

```diff
@@ -297,12 +297,16 @@ export default withRouter(
 								emptyState={
 									!search && !filter ? (
 										<ClayEmptyState
-											description="There are no questions inside this topic. Be the first to ask something!"
+											description={Liferay.Language.get(
+												'there-are-no-questions-inside-this-topic-be-the-first-to-ask-something'
+											)}
 											imgSrc={
 												context.includeContextPath +
 												'/assets/empty_questions_list.png'
 											}
-											title="This topic is empty."
+											title={Liferay.Language.get(
+												'this-topic-is-empty'
```

Comment:
This should not be tthere-are-no-questions-inside-this-topic-be-the-first-to-ask-something? What is the difference?

Reply:
One is the title and the other is the description of the empty state. The image is more descriptive
<img width="380" alt="imagen" src="https://user-images.githubusercontent.com/17163902/93597409-4d577800-f9bb-11ea-92ef-e73d860300ba.png">
For us, each one is a different message inside the empty state.

Reply:
Understood! Sorry for the confusion